test/code_block_indented.md

```diff
@@ -14,4 +14,10 @@ But we'll do another:
     And this
     will.
 
+One last one:
+
+    This is code
+    
+    with an empty line.
```

Comment:
“empty indented line”

Reply:
One more thing: Could you please add an additional test case to each file with the “opposite“ scenario? In addition to verifying the rule now allows what we want, I want to be sure that it continues not allowing what we don’t want. Thank you!

Reply:
Not sure I understand what you mean by "each file". And what would be the opposite for the indented blocks?

I updated the commit, and added opposite tests for the fenced block (see below), and the BR trailing spaces.

Reply:
I meant both test files modified by the commit. But it sounds like you you the drift in your comments above. I’ll have a look at the next iteration and let you know it there’s anything else.